app/helpers/job_invocation_output_helper.rb

```diff
@@ -1,20 +1,20 @@
 module JobInvocationOutputHelper
   CONSOLE_COLOR = {
-      '31' => 'red',
-      '32' => 'lightgreen',
-      '33' => 'orange',
-      '34' => 'deepskyblue',
-      '35' => 'mediumpurple',
-      '36' => 'cyan',
-      '37' => 'grey',
-      '91' => 'red',
-      '92' => 'lightgreen',
-      '93' => 'yellow',
-      '94' => 'lightblue',
-      '95' => 'violet',
-      '96' => 'turquoise',
-      '0'  => 'default',
-    }
+    '31' => 'red',
+    '32' => 'lightgreen',
+    '33' => 'orange',
+    '34' => 'deepskyblue',
+    '35' => 'mediumpurple',
+    '36' => 'cyan',
+    '37' => 'grey',
+    '91' => 'red',
+    '92' => 'lightgreen',
+    '93' => 'yellow',
+    '94' => 'lightblue',
+    '95' => 'violet',
+    '96' => 'turquoise',
+    '0'  => 'default',
+  }.freeze
```

Comment:
This won't work because of the '.default' unfortunately.. I've fixed this in #130 with https://github.com/theforeman/foreman_remote_execution/pull/130/files#r165339121

Reply:
I think there's a better way while keeping the default being set and not copying the default value out of it, let's see if the updated version works